2023/incoming/trimming-audio-files-with-audio-nama.pod

```diff
@@ -0,0 +1,475 @@
+Author: Joel Roth <joelz@pobox.com>
```

Comment:
Can we remove this entire file from the branch?

Reply:
Yes.  I assume this is done since the PR has been closed.